src/AbstractArray.php

```diff
@@ -491,31 +485,21 @@ public function getRandom()
     /**
      * Pick a given number of random keys/indexes out of this array.
      *
-     * @param int $number The number of keys/indexes (should be > 1 and < $this->count())
+     * @param int $number The number of keys/indexes (should be <= $this->count())
      *
-     * @return array Random keys of array
+     * @return mixed Random keys or key of array
      *
      * @throws \RangeException
-     * @throws \LogicException
      */
     public function getRandomKeys($number)
     {
         $number = (int) $number;
         $count = $this->count();
-        if ($number < 1 || $number > $count) {
-            throw new \RangeException(sprintf(''
-                . 'The given number "%d" should be greater than or equal to "1" '
-                . 'and less than or equal to "%d" (the number of elements in the array).',
-                $number,
-                $count
-            ));
-        }
-        if ($number === 1 || $number === $count) {
-            throw new \LogicException(sprintf(
-                'The given number "%d" should not be equal to "1" or "%d" (the number of elements in the array).',
+        if (!$number || $number === 0 ||$number > $count) {
+            throw new \RangeException(sprintf(
+                'Number of requested keys (%s) must be lower or equal than available array elements (%s)',
```

Comment:
Could we change to `... must be equal or lower than number of elements in this array (%s)`?